vnext/Directory.Build.props

```diff
@@ -8,33 +8,30 @@
     <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
 
     <ReactNativeWindowsDir>$(MSBuildThisFileDirectory)</ReactNativeWindowsDir>
+    <FollyDir>$(ReactNativeWindowsDir)node_modules\react-native\folly</FollyDir>
+    <ReactNativeDir>$(ReactNativeWindowsDir)node_modules\react-native</ReactNativeDir>
+    <YogaDir>$(ReactNativeWindowsDir)node_modules\react-native\ReactCommon\yoga</YogaDir>
   </PropertyGroup>
 
   <PropertyGroup Label="Configuration">
-    <PlatformToolset>v141</PlatformToolset>
     <ProjectName Condition="'$(ProjectName)'==''">$(MSBuildProjectName)</ProjectName>
 
     <!-- Visual Studio forces using 'Win32' for the 'x86' platform. -->
-    <!-- NOTE: This assignment of `Platform` won't persist thourgh the build. -->
-    <Platform Condition="'$(Platform)'=='Win32'">x86</Platform>
-    <BaseIntDir>$(ReactNativeWindowsDir)build\$(Platform)\$(Configuration)</BaseIntDir>
-    <BaseOutDir>$(ReactNativeWindowsDir)target\$(Platform)\$(Configuration)</BaseOutDir>
+    <BaseIntDir Condition="'$(Platform)' != 'Win32'">$(ReactNativeWindowsDir)build\$(Platform)\$(Configuration)</BaseIntDir>
+    <BaseIntDir Condition="'$(Platform)' == 'Win32'">$(ReactNativeWindowsDir)build\x86\$(Configuration)</BaseIntDir>
+    <BaseOutDir Condition="'$(Platform)' != 'Win32'">$(ReactNativeWindowsDir)target\$(Platform)\$(Configuration)</BaseOutDir>
+    <BaseOutDir Condition="'$(Platform)' == 'Win32'">$(ReactNativeWindowsDir)target\x86\$(Configuration)</BaseOutDir>
 
-    <!-- Windows Desktop -->
     <IntDir>$(BaseIntDir)\$(ProjectName)\</IntDir>
     <OutDir>$(BaseOutDir)\$(ProjectName)\</OutDir>
-  </PropertyGroup>
 
-  <PropertyGroup>
-    <FollyDir>$(ReactNativeWindowsDir)node_modules\react-native\folly</FollyDir>
-    <ReactNativeDir>$(ReactNativeWindowsDir)node_modules\react-native</ReactNativeDir>
-    <YogaDir>$(ReactNativeWindowsDir)node_modules\react-native\ReactCommon\yoga</YogaDir>
+    <GeneratedFilesDir>$(IntDir)Generated Files\</GeneratedFilesDir>
```

Comment:
>$(IntDir) [](start = 23, length = 9)

are directory related variables suppose to contain backslash character '\' at the end?
BaseIntDir doesn't have backslash, but IntDir does.
Please make them consistent.

Reply:
The inconsistency comes from MSBuild conventions themselves.
In particular, `SolutionDir` is enforced to have a trailing slash.
Other variables, not necessarily.
(If curious, you can look at the VCTargets path where these variables are defined).